benchmark/Cabana_CommPerformance.cpp

```diff
@@ -445,14 +445,11 @@ int main( int argc, char* argv[] )
 #ifdef KOKKOS_ENABLE_CUDA
     using CudaDevice = Kokkos::Device<Kokkos::Cuda, Kokkos::CudaSpace>;
     using CudaUVMDevice = Kokkos::Device<Kokkos::Cuda, Kokkos::CudaUVMSpace>;
+    using HostDevice = Kokkos::Device<Kokkos::Serial, Kokkos::HostSpace>;
```

Comment:
```suggestion
    using HostDevice = Kokkos::DefaultHostExecutionSpace::device_type;
```
so it works even in case the serial backend is not enabled